src/org/labkey/test/tests/AbstractExportTest.java

```diff
@@ -499,6 +511,12 @@ protected final void assertPerlScriptContents(String perlScript, String filterCo
             assertTrue("Script is missing filterArray property", perlScript.contains("['" + filterColumn + "', eq, ''foo'']"));
     }
 
+    protected final void assertStableURLContents(String stableURL)
+    {
+        assertTrue("Script is missing title", stableURL.contains("The following URL can be used to reload the query, preserving any filters, sorts, or custom sets of columns:"));
+        assertTrue("Script is missing URL", stableURL.contains("query-executeQuery.view?"));
```

Comment:
Let's validate more of the URL. I think everything after the query-executeQuery.view? should be stable and consistent too